package/MDAnalysis/analysis/base.py

```diff
@@ -40,17 +40,60 @@
 logger = logging.getLogger(__name__)
 
 
+class Results(dict):
+    r"""Container object for storing results.
+
+    Results are extend dictionaries by enabling values to be accessed by key,
+    `results["value_key"]`, or by an attribute, `results.value_key`. 
+    They store all results obatined from an analysis after the ``run`` call.
+
+    The current implementation is similar to the `Bunch` class in `sklearn`.
```

Comment:
double back ticks, and you can use
```suggestion
    The current implementation is similar to the :class:`sklearn.utils.Bunch` class in `scikit-learn`_.
  
    .. _`scikit-learn`: https://scikit-learn.org/
     
```